libs/dh/admin/data-access-api/src/lib/dh-admin-user-management-data-access-api.store.ts

```diff
@@ -81,12 +95,30 @@ export class DhAdminUserManagementDataAccessApiStore extends ComponentStore<DhUs
           .pipe(
             tapResponse(
               (response) => {
-                this.setLoading(LoadingState.LOADED);
+                if (state.hasInitialPageLoaded === false) {
```

Comment:
98..107 and 112..121 seem identical and could be moved to a function called from both places